src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -95,13 +96,13 @@ export class GroupInviteUsersComponent {
   }
 
   onInviteClicked(): void {
-    if (!this.textArea || !this.group) return;
+    if (!this.group) return;
```

Comment:
Aren't you supposed to check that the form is valid somehow?

Reply:
And where is the validation BTW? Isn't it supposed to be "moved" to reactive form?